orangecontrib/text/widgets/owtwitter.py

```diff
@@ -0,0 +1,275 @@
+import functools
+
+from PyQt4 import QtGui, QtCore
+from PyQt4.QtGui import QComboBox, QDialog
+
+from Orange.widgets import gui
+from Orange.widgets.settings import Setting
+from Orange.widgets.widget import OWWidget
+
+from orangecontrib.text import twitter
+from orangecontrib.text.corpus import Corpus
+from orangecontrib.text.widgets.ownyt import _i
+from orangecontrib.text.languge_codes import lang2code
+
+
+class APICredentialsDialog(QDialog):
+    size_hint = (470, 100)
```

Comment:
This seems to be just a few pixels to narrow for my key. The first letter of the secret is partially hidden. Since I suppose you set the width so that whole inputs could be seen without scrolling, I suggest you make it a bit wider.
![unknown](https://cloud.githubusercontent.com/assets/713026/15654548/85bd8b9c-2695-11e6-80e4-b561d4b11eed.jpeg)